feegrant/index.md

```diff
@@ -0,0 +1,239 @@
+---
+parent:
+  title: Understanding the Fee Grant Module
+order: 0
+description: Use the Cosmos SDK feegrant module to grant the use of tokens to pay for fees from one account (the granter) to another account (the grantee).
+---
+
+# Feegrant Module
+
+The [feegrant](https://docs.cosmos.network/v0.44/modules/feegrant/) enables a user / contract / module (the granter) to pay the fees for someone else (the grantee) when they want to broadcast a transaction on the blockchain. A key feature of feegrant is that the granter retains full access to their tokens, and is able to revoke the allowance at any time.
```

Comment:
```suggestion
# Fee Grant Module

The [fee grant](https://docs.cosmos.network/v0.44/modules/feegrant/) module enables the granted (a user, contract, or module) to pay the fees for someone else (the grantee) when the grantee wants to broadcast a transaction on the blockchain. A key feature of fee grant is for the granter to retain full access to their tokens and be able to revoke the allowance at any time.
```

Reply:
Several points here:

- fee grant = two words (I didn't mark the rest, please update all)
- avoid "they" and instead repeat the subject (I guessed at "the grantee" for "they" in this suggested update)
- do not use slashes to separate alternatives, see https://developers.google.com/style/slashes?hl=en

Reply:
@nooomski I was trying to repair a painful sentence... Let's try this 

> The granter retains full access to their tokens and is able to revoke the allowance at any time.

Reply:
and yes "granted" was my human error, thanks for catching that